### PR TITLE
SFR-1351 Migrate Feature Branch Deployments to TugboatQA

### PR DESCRIPTION
**Pull request by mwbenowitz**
This removes the custom AWS ECS feature branch deployment in favor of a standard implementation that can be shared by NYPL Digital. This uses the Tugboat.qa service to deploy and manage the preview builds. This commit simply swaps out these build tools.

[Jira Ticket](http://jira.nypl.org/browse/SFR-1351)

### CODE REVIEW

**Comment by hokei** [comment]
Is there a way to have tugboat preview this deployment on this PR or does it need to be merged first?  
(Where is it being deployed?)

File: .tugboat/config.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  drb-front-end:
+    image: tugboatqa/node:16
+    default: true
+    expose: 3000
+    commands:
+      build:
+        - npm install
+        - npm run build
+      start:
+        - npm run start &
```
Comment (hokei) [inline .tugboat/config.yml:11]: `start &`  ?  (It looks wrong but what do I know)

Comment (mwbenowitz) [reply .tugboat/config.yml:11]: It _is_ weird but it's a hack that Tugboat uses to get a service running in the background. Without it Tugboat thinks the start command hangs and never gives you the preview URL.

I don't like it but it's in their documentation!

**Comment by mwbenowitz** [comment]
It should post a deployment status here but I didn't have that option enabled when I opened the PR, so hopefully it should do it next time, it definitely is deployed though! https://pr301-9g2jmajzau5ndwwgz5nlb7kubkikbtzc.tugboat.qa/

**Review by hokei — APPROVED** [review]
OK let's get this merged in so we can make a base image!

**Pull request merged**